docker-compose.yml

```diff
@@ -1,4 +1,36 @@
 services:
+  zookeeper:
+    image: zookeeper:3.8
+    volumes:
+      - zkdata:/data
+      - zkdatalog:/datalog
+    ports:
+      - 2181:2181
+
+  kafka:
+    image: wurstmeister/kafka:2.13-2.8.1 # This tag is <scala version>-<kafka version>
+    volumes:
+      - kafkadata:/kafka/kafka-logs-kafka
+    ports:
+      - 9092:9092
+    hostname: kafka
+    depends_on:
+      - zookeeper
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      KAFKA_ADVERTISED_LISTENERS: INSIDE://:29092,OUTSIDE://localhost:9092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT
+      KAFKA_LISTENERS: INSIDE://:29092,OUTSIDE://:9092
+      KAFKA_INTER_BROKER_LISTENER_NAME: INSIDE
+
+  redpanda:
+    image: docker.redpanda.com/vectorized/console:master-cb51d72
+    depends_on:
+      - kafka
```

Comment:
We may want to put this same depends on `eventsapi` since now it implicitly expects `kafka` to be there when it starts up.